CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 ﻿# Change Log
 
 <!--## Unreleased-->
+## v2.10.1
 
+- Fix issue there're some applications that launched process exists immediately(Ex: microsoft calculator, firefox, chrome, iexplorer), so in close/quit function it throws exception Process Not Found.

```

Comment:
@quangnt Please make this brief just like the author is doing like https://github.com/PersonifyInc/Winium.Cruciatus/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12 and here https://github.com/PersonifyInc/Winium.Cruciatus/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR17